pyfakefs/fake_pathlib.py

```diff
@@ -519,6 +518,9 @@ class _FakePosixFlavour(_FakeFlavour):
         independent of FakeFilesystem properties.
         """
 
+        sep = "/"
+        altsep = ""
```

Comment:
This catches my eye -- elsewhere (like in `helpers.py`) this is `None`, not an empty string. Is the empty string correct here?

Reply:
I was just lazy - this way I didn't need to add typing information, and these are set during intializing anyway. But you are right, better to do this consistently.